robot/rover/MobilePlatform/commands.h

```diff
@@ -666,4 +697,29 @@ void Commands::stop(bool timeout) {
 
 }
 
+void Commands::ledOn(void){
+    ledStatus = true;
+}
+
+void Commands::ledOff(void){
+    ledStatus = false;
+}
+
+void Commands::ledManCtrl(void){
+    ledColor[0] = 0; // Blue
+    ledColor[1] = 0;
+    ledColor[2] = 255;
+}
+
+void Commands::ledAutoCtrl(void){
+    ledColor[0] = 255; // Red
+    ledColor[1] = 0;
+    ledColor[2] = 0;
+}
+
+void Commands::ledEureka(void){
+    ledColor[0] = 0; // Green
+    ledColor[1] = 255;
+    ledColor[2] = 0;
```

Comment:
I agree, I will discuss with Peter or Cedric possible solutions they have. My initial solution to try and get around doing this was to handle setting the LED colors in the Commands class but thought it best to avoid that and handle it all in the main loop.

Thank you for all the feedback, every bit is appreciated.